### PR TITLE
Handle missing values in metadata component. Closes #126

### LINKED ISSUE #126: Handle missing values in Preview for Metadata component
### Motivation

As was described in #115, there is insufficient check for missing values for `Metadata` component values.

Great jop @Karl-EdwardFPJeanMehu, thanks!
### Proposed solution

According to the metadata (i.e. ogImage) I would recommend using the lodash for the value check:
```
import _ from 'lodash';
...
      {!_.has(props, 'ogImage.value[0].url') ? null : (  // current state =>  {!props.ogImage ? null : (
        <meta property=\"og:image\" content={props.ogImage.value[0].url} />
      )}
```

### PR DESCRIPTION
**Pull request by Karl-EdwardFPJeanMehu**
This PR solves missing values in the Metadata component. Closes #126

### CODE REVIEW

File: src/Components/Metadata.js [hunk 1]
```diff
@@ -1,54 +1,57 @@
...
         <meta name="description" content={props.description.value.trim()} />
       )}
 
-      {!props.ogTitle ? null : (
+      {!_.has(props, 'ogTitle.value') ? null : (
         <meta property="og:title" content={props.ogTitle.value.trim()} />
       )}
-      {!props.ogTitle ? null : <meta property="og:type" content="website" />}
-      {!props.ogTitle ? null : (
+      {!_.has(props, 'ogTitle.value') ? null : (
```
Comment (Simply007) [inline src/Components/Metadata.js]: Could you merge these two checks or ogTitle and ogType to one code block?

File: src/Components/Metadata.js [hunk 2]
```diff
@@ -1,54 +1,57 @@
...
+        <meta property="og:type" content="website" />
+      )}
+      {!_.has(props, 'ogTitle.value') ? null : (
         <meta property="og:url" content={window.location.href} />
       )}
-      {!props.ogImage ? null : (
+      {!_.has(props, 'ogImage.value[0].url') ? null : (
         <meta property="og:image" content={props.ogImage.value[0].url} />
       )}
       {!props.ogDescription ? null : (
```
Comment (Simply007) [inline src/Components/Metadata.js]: Could you implement check also for ogDescription for empty string?

**Comment by Karl-EdwardFPJeanMehu** [comment]
No problem

**Comment by Simply007** [comment]
Sorry @Karl-EdwardFPJeanMehu for that long approval. I have completely missed the pull request..

**Pull request merged**